script.js

```diff
@@ -0,0 +1,196 @@
+// speaker card
+function speaker(photo, Name, Work, Descrition, containerC) {
+  // Creation of elements
+  const AllSpeakers = containerC;
+  const speakerWrapper = document.createElement('section');
+  const speakerImage = document.createElement('div');
+  const behindPicture = document.createElement('div');
+  const SpeakerDescription = document.createElement('div');
+  const profileImage = document.createElement('img');
+  const speakerName = document.createElement('h4');
+  const SpeakerWork = document.createElement('h6');
+  const DescritionText = document.createElement('p');
+  const hr = document.createElement('hr');
+
+  // Assingment of selectors
+  speakerWrapper.classList.add('speaker-wrapper');
+  speakerImage.classList.add('speaker-image');
+  behindPicture.classList.add('behind-picture');
+  SpeakerDescription.classList.add('speaker-description');
+  profileImage.classList.add('image');
+  speakerName.classList.add('speaker-name');
+  SpeakerWork.classList.add('inline2');
+  DescritionText.classList.add('description-text');
+
+  // Tree organization
+  AllSpeakers.append(speakerWrapper);
+  speakerWrapper.append(speakerImage, SpeakerDescription);
+  speakerImage.append(behindPicture, profileImage);
+  SpeakerDescription.append(speakerName, SpeakerWork, hr, DescritionText);
+
+  // value definition
+  profileImage.src = `${photo}`;
+  speakerName.innerText = Name;
+  SpeakerWork.innerText = Work;
+  DescritionText.innerText = Descrition;
+}
+
+const Data1 = [{
+  photo: './IMG/KARIM.jpeg',
+  Name: 'Mr Karim Abdelghani',
+  Work: 'ITU Regional Officer for the Arab States',
+  Descrition: 'With an experience of around 17 years in Information and Communication Technology',
+}, {
+  photo: './IMG/speakers/IMG_8237.jpg',
+  Name: 'Mr Ushindi Gedeon',
+  Work: 'Software Developer',
+  Descrition: 'Co-founder of RX-telecom, With an experience in Information and Communication Technology',
+}, {
+  photo: './IMG/speakers/AmadouGarba.jpg',
+  Name: 'Mr Karim Abdelghani',
+  Work: 'Senior (Technology) Coordinator',
+  Descrition: 'Associate Professor at Carnegie Mellon University, Africa and Director General of the Multi-Sector Regulatory Authority of Niger',
+}, {
+  photo: './IMG/speakers/BEREAUX-Stephen.jpg',
+  Name: 'Mr Stephen Bereaux',
+  Work: 'Deputy Director, ITU Development Bureau ',
+  Descrition: 'aith more than 20 years of senior management experience in regulatory bodies in the Caribbean region.',
+}, {
+  photo: './IMG/speakers/AMINATA.JPG',
+  Name: 'Ms Mame Aminata Diambo',
+  Work: 'Student',
+  Descrition: 'Student, ESMT Dakar, Senegal',
+}, {
+  photo: './IMG/speakers/allo-cin2.jpg',
+  Name: 'Mr Lacina Koné',
+  Work: 'CEO of Smart Africa',
+  Descrition: ' Advisor to the Prime Minister of the Republic of Côte d`Ivoire (2017-2018), in charge of Digital Transformation',
+}];
+
```

Comment:
- [ ] Kindly make sure to add the corner checks background image to all the speakers. The background image is shown in the video you recorded but it is not visible on the Github page you deployed nor when I clone it on my mac.
- [ ] Kindly also follow the spacing and give the appropriate margin as per the original design.
- Original design:
<img width="830" alt="Screenshot 2022-04-15 at 3 21 32 PM" src="https://user-images.githubusercontent.com/70269507/163570094-5400c2b8-eead-4e63-8be9-2d82682f0e60.png">

- Your design:
<img width="1268" alt="Screenshot 2022-04-15 at 3 21 11 PM" src="https://user-images.githubusercontent.com/70269507/163570084-dfa8775b-1660-46b0-9129-13b1349d98c0.png">